Stiletto.Core/StilettoMakerGUI.cs

```diff
@@ -174,25 +174,36 @@ private void RegisterMakerControls(Stiletto plugin, RegisterSubCategoriesEvent e
 
         public void OnHeelInfoUpdate(object _, HeelInfoEventArgs heelInfo)
         {
-            if (MakerAPI.InsideMaker && heelInfo != null)
+            try
             {
-                if (slider_Height != null)
+                // When using Maker for a second time in a game session there is a
+                // [Error  :Null Checks] Swallowing exception to prevent game crash!
+                // This occurs when accessing 'heelInfo' that is disposed of.
+                // After RegisterMakerControls is called there are no more exceptions.
+                if (MakerAPI.InsideMaker && heelInfo != null)
```

Comment:
Wouldn't using InsideAndLoaded fix this issue? A blank catch is not great because it will hide unexpected issues, at least log the exceptions once caught.

You can also use `MakerAPI.MakerExiting` to clean up any hooks or fields that were set when entering maker so that you return to the clean initial state.

Reply:
Yes this does fix the issue.